rholang/src/test/scala/coop/rchain/rholang/interpreter/NormalizeTest.scala

```diff
@@ -728,6 +730,34 @@ class ProcMatcherSpec extends FlatSpec with Matchers {
     result.par should be(expectedResult)
     result.knownFree should be(inputs.knownFree)
   }
+
+  "PBundle" should "normalize terms inside a bundle" in {
+    val pbundle     = new PBundle(new PVar(new ProcVarVar("x")))
+    val boundInputs = inputs.copy(env = inputs.env.newBinding(("x", ProcSort, 0, 0)))
+
+    val result = ProcNormalizeMatcher.normalizeMatch(pbundle, boundInputs)
+
+    val expectedResult =
+      inputs.par
+        .withBundles(List(Bundle(EVar(BoundVar(0)))))
+        .withLocallyFree(BitSet(0))
+
+    result.par should be(expectedResult)
+    result.knownFree should be(inputs.knownFree)
+  }
+
+  /** Example:
+    * bundle { _ | x }
+    */
+  it should "not count free variable and the wildcard from the body" in {
```

Comment:
The title of this test should change.

Reply:
Done

Reply:
Fixed error message. Currently for this specific case it throws:
```
coop.rchain.rholang.interpreter.NormalizerExceptions$UnexpectedBundleContent: Bundle's content shouldn't have free variables or wildcards.  Wildcards at positions: line: 0, column: 0. Free variables at positions: line: 0, column: 0.
```

I don't understand why there are only `0` -s though. Can you help? @guardbotmk3